tests/test_1072_v2-sort.py

```diff
@@ -69,14 +77,21 @@ def test_masked():
         [1, 2, None, None, None],
     ]
```

Comment:
Do you know why it's saying "'ListOffsetArray' has no 'mask'"? That's a bad sign, that some code thinks it's working with a ByteMaskedArray (or less likely, a BitMaskedArray), but it actually has a ListOffsetArray.

At first, I thought this was a regression, since the `pytest.mark.skip` is new, but now I see that it's replacing a commented-out version, which is always better. At least we'll see the fact that there's something wrong here in the list of skipped tests.

Reply:
I think, `mask` suppose to be applied at a high level `ak.Array`, not a layout. I left the test as a reminder to check it after we promote v2 to a high level. I’ll check the layout before I do a v1 to v2 conversion.

Reply:
If any object is passed into the `ak.mask` function, `ak.to_layout` will attempt to convert it into a layout, which means stripping off an `ak.Array` if that's what the object is, a no-op if it's already a layout, and a conversion if it's something else. However, it might not be able to do the conversion if it's a v2 layout (i.e. if `v1_to_v2` is not called in `ak.to_layout`).